fundamentals/impl.tex

```diff
@@ -770,29 +732,6 @@ \subsubsection{Version Control}
 it is possible for \Cyclus to accurately replay the entirety of who did what to the
 code when.
 
-\Cyclus uses a strategy known as \emph{git flow}
```

Comment:
similar to above. I think the figure is helpful, but the words are too many. I'll revert this, but submit a change that pares down this section.